src/modules/auth/dto/auth.dto.ts

```diff
@@ -0,0 +1,47 @@
+import { ApiProperty } from '@nestjs/swagger';
+import { IsString } from 'class-validator';
+export class SignUpUserBody {
+  @ApiProperty()
+  @IsString()
+  adminId: string;
```

Comment:
can add these for the required fields
  @IsNotEmpty({ message: 'Admin Id is required' })